src/clojush/pushgp/breed.clj

```diff
@@ -82,23 +82,39 @@
              rand-gen
              argmap))))
 
+(defn update-instruction-map-uuids
+  "Takes an individual and updates the UUIDs on every instruction-map in its
+   :geneome, except for the ones which are a random insertion."
+  [individual]
+  (update individual :genome
+          (fn [genome]
+            (mapv (fn [instruction-map]
+                   (if (:random-insertion instruction-map)
+                     (dissoc instruction-map :random-insertion)
```

Comment:
Why do we remove that entry? Do we just use the _absence_ of `:parent-uuid` to indicate that it's been randomly generated?

Any how does a randomly inserted gene get it's UUID? Does that happen somewhere else? I'm pretty sure it does, but it's not obvious here so I wanted to ask about it.